### DISCUSSION TITLE
after upgrade to 1.24.1 repeated unrecognized address error

### DISCUSSION THREAD

**Discussion by dagda1** [denoland/deno#15396] [Q&A]
After upgrading to deno 1.24.1 via homebrew on a mac m1, I repeatedly get this error

> assertion failed [block != nullptr]: BasicBlock requested for unrecognized address

I have to restart the terminal session for it to go away and then it comes back

**Comment by zachauten** [comment]
What command are you running? Do you get this issue from a deno binary downloaded directly?

**Comment by crucialfelix** [comment]
I am getting this too. It's caused by something involving `--allow-net` requests.

I am just fetching some http responses, and at some point it gets stuck. Maybe I threw an error during development and something doesn't get closed.

The terminal session is then infected with this.  Restarting terminal session clears it up.
